main.go

```diff
@@ -194,17 +208,6 @@ var mainInstance = proxy{
 
 func init() {
 	flag.StringVar(&mainInstance.flags.configFileName, "configfile", "sf/sfdbproxy.conf", "Name of the db proxy configuration file")
-	flag.StringVar(&mainInstance.etcdMgr.operation, "cluster-op", "", "operation to perform if running in cluster mode [\"seed\", \"join\", \"\"]")
```

Comment:
we'll need to override this on the commandline to seed the initial machine won't we? How else would we start the cluster.